axelor-base/src/main/resources/apps/demo-data/en/base_addressTemplate.csv

```diff
@@ -0,0 +1,2 @@
+"importId";"name";"engineSelect";"templateStr"
+"1";"DEFAULT";"2";"${(Address.addressL2 == null) ? '' : Address.addressL2} \n ${(Address.addressL3 == null) ? '' : Address.addressL3} \n ${(Address.addressL4 == null) ? '' : Address.addressL4} \n ${(Address.addressL5 == null) ? '' : Address.addressL5} \n  ${(Address.addressL6 == null) ? '' : Address.addressL6} \n ${(Address.addressL7Country == null) ? '' : Address.addressL7Country.name}"
```

Comment:
"\n" is not correct, when I check the address template in the app, I see this char. I don't want to see it, when I open the address template I want to see all the fields on a different line.
Also when I import the demo data, the formattedFullName is not computed on any address